### PR TITLE
Tlunizin 767 - add test script for canvasTLAdmin ESB API

### PR DESCRIPTION
**Pull request by dlhaines**

### CODE REVIEW

File: ApiTests/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Have git ignore a file with real credentials.
```
Comment (zqian) [inline ApiTests/.gitignore]: .gitignore does not need to be checked in.

File: ApiTests/canvasTLAdminTest.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+### Set the token and prefix values for the different users.
+source ./AUTH.sh
+
+help
+
+echo "++++++++++ testing canvas access $CHANNEL_NAME ++++++++++"
+
+function run_tests {
+    echo "############################# $CHANNEL_NAME ###################"
+#    echo "+++ trying self $CHANNEL_NAME +++"
```
Comment (zqian) [inline ApiTests/canvasTLAdminTest.sh]: remove the commented lines?

File: ApiTests/.gitignore (hunk 1 above)
Comment (dlhaines) [inline ApiTests/.gitignore]: Yes it does.  It prevents someone from checking in AUTH.sh by accident.

File: ApiTests/canvasTLAdminTest.sh (hunk 2 above)
Comment (dlhaines) [inline ApiTests/canvasTLAdminTest.sh]: No. These are examples of different queries that might be useful, so they should stay as reference.  The style could be clearer but I'd like to see how much this code is used before more time is put into it.

**Pull request merged**